test/e2e/webhook_e2e_test.go

```diff
@@ -0,0 +1,955 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	. "github.com/onsi/ginkgo"
+	"github.com/stretchr/testify/require"
+	admissionregistrationv1 "k8s.io/api/admissionregistration/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/operator-framework/api/pkg/operators/v1alpha1"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/controller/install"
+	"github.com/operator-framework/operator-lifecycle-manager/pkg/lib/operatorclient"
+)
+
+var _ = Describe("Webhook", func() {
+	It("creates global webhooks", func() {
+		defer cleaner.NotifyTestComplete(GinkgoT(), true)
+
+		c := newKubeClient(GinkgoT())
+		crc := newCRClient(GinkgoT())
+
+		namespace, nsCleanupFunc := newNamespace(GinkgoT(), c, genName("csc-test-"))
+		defer nsCleanupFunc()
+
+		og := newOperatorGroup(namespace.Name, genName("test-og-"), nil, nil, []string{}, false)
+		og, err := crc.OperatorsV1().OperatorGroups(namespace.Name).Create(context.TODO(), og, metav1.CreateOptions{})
+		require.NoError(GinkgoT(), err)
```

Comment:
could be replaced with Gomega's matcher: `Expect(err).ToNot(HaveOccurred())`
As we are transitioning towards ginkgo, the idea is to stray away from `/testify/require` package and use gomega assertions instead